django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
Yeah on second thought I agree. The security benefit of moving a better hasher that requires a third-party install to the top of the default list is minimal or nonexistent, since you still have to explicitly install the third-party requirement to gain the benefit, at which point you may as well have to set the setting as well. Automatic changes in behavior based only on the presence or absence of a third-party package are questionable in general.